benches/lib.rs

```diff
@@ -70,3 +77,71 @@ fn bench_lib_merge_results(b: &mut Bencher) {
 
     b.iter(|| black_box(grcov::merge_results(&mut result, result2.clone())));
 }
+
+#[bench]
+fn bench_lib_consumer(b: &mut Bencher) {
+    let num_threads = 2;
+    let result_map: Arc<SyncCovResultMap> = Arc::new(Mutex::new(FxHashMap::with_capacity_and_hasher(20_000, Default::default())));
+    let (sender, receiver) = unbounded();
+    let source_root = None;
+    let working_dir = PathBuf::from("");
+    let gcno_buf: Vec<u8> = vec![
```

Comment:
You could read a gcno file, so you don't have to hardcode the bytes here

Reply:
I could. I copy/paste this gcno from a producer test, I think it's short enough to be inlined.